org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -65,9 +65,28 @@ define([
             formData: [{
               entity_table: customSettings.entityTable || error('entityTable'),
               crm_attachment_token: customSettings.crmAttachmentToken || error('crmAttachmentToken')
+            }],
+            filters: [{
+              name: 'fileFormatFilter',
+              fn: function (item /*{File|FileLikeObject}*/ , options) {
+                var type = '|' + item.type.slice(item.type.lastIndexOf('/') + 1) + '|';
+
+                //checks for mime types and not file name extensions
```

Comment:
why do you need to check for mime types? There is no mention of this in the technical details